shell/src/main/java/alluxio/cli/fsadmin/FileSystemAdminShellUtils.java

```diff
@@ -47,32 +48,22 @@ public static int compareTierNames(String a, String b) {
   }
 
   /**
-   * @return whether master client service is running
+   * Checks if the master client service is running.
    */
-  public static boolean masterClientServiceIsRunning() {
-    // Check if Alluxio master and client services are running
+  public static void masterClientServiceIsRunning() throws IOException {
     try (CloseableResource<FileSystemMasterClient> client =
         FileSystemContext.INSTANCE.acquireMasterClientResource()) {
-      MasterInquireClient inquireClient = null;
       try {
         InetSocketAddress address = client.get().getAddress();
         List<InetSocketAddress> addresses = Arrays.asList(address);
-        inquireClient = new PollingMasterInquireClient(addresses, () ->
+        MasterInquireClient inquireClient = new PollingMasterInquireClient(addresses, () ->
             new ExponentialBackoffRetry(50, 100, 2));
-      } catch (UnavailableException e) {
-        System.err.println("Failed to get the leader master.");
-        System.err.println("Please check your Alluxio master status");
```

Comment:
should we keep the handling for failing to determine leader vs failing to make a leader request?

Reply:
Changed to keep both, thanks!